geometry_msgs/PoseStamped pose
int8 marker_name